docs/user-guide/03-oscap-remediation.md

```diff
@@ -13,13 +13,13 @@ boot of the image.
 
 ## Build-time Remediation
 
-To solve this issue, an [osbuild stage] runs the `OpenSCAP` tool on the filesystem tree while the image is being built. The `OpenSCAP` tool runs 
+To solve that limitation, `osbuild-composer` uses the build-time Remediation: an [osbuild stage] runs the `OpenSCAP` tool to search for vulnerabilities on the filesystem tree while the image is being built. The `OpenSCAP` tool runs 
```

Comment:
```suggestion
To solve that limitation, `osbuild-composer` uses the build-time remediation: an [osbuild stage] runs the `OpenSCAP` tool to search for vulnerabilities on the filesystem tree while the image is being built. The `OpenSCAP` tool runs 
```

Why is "remediation" capitalized?

Reply:
Initially to mark the tool. But I noticed that it should be capitalized when referring  to  the Remediation online tool, so it was a mistake. Thanks for noticing it.